### PR TITLE
template for writing installation recipes

### PR DESCRIPTION
**Pull request by jemten**

### CODE REVIEW

File: templates/code/recipe_install_shell.pm [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+## Constants
+Readonly my $DOT     => q{.};
+Readonly my $NEWLINE => qq{\n};
+Readonly my $SPACE   => q{ };
+
+#############################################################################
+############### SHORT INSTRUCTIONS ON HOW TO USE THE TEMPLATE ###############
+#############################################################################
+# This is a genereric template for writing installation commands for
+# installation of progrmas via SHELL. The installation proccess required by
```
Comment (henrikstranneheim) [inline templates/code/recipe_install_shell.pm]: progrmas -> programs

File: templates/code/recipe_install_shell.pm [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+                env_name   => $conda_environment,
+                FILEHANDLE => $FILEHANDLE,
+            }
+        );
+        say $FILEHANDLE $NEWLINE;
+    }
+
+    ## Download
+    say {$FILEHANDLE} q{## Download PROGRAM};
+    my $url = q{https://                 };
```
Comment (henrikstranneheim) [inline templates/code/recipe_install_shell.pm]: What happened here? Should there be that many whitespace chars?

File: templates/code/recipe_install_shell.pm [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+            {
+                FILEHANDLE => $FILEHANDLE,
+            }
+        );
+        say $FILEHANDLE $NEWLINE;
+    }
+
+    print {$FILEHANDLE} $NEWLINE;
+    return;
+
```
Comment (henrikstranneheim) [inline templates/code/recipe_install_shell.pm]: Remove empty line

**Comment by henrikstranneheim** [comment]
Good on ya mate! Don't ya forget the eskie and togs when your having a barbie at the beach celebrating christmas!

**Comment by jemten** [comment]
Haha! What jibberish is that :)

**Comment by henrikstranneheim** [comment]
Here is a tip for the trip: https://www.ef.com/blog/language/10-aussie-expressions-to-master/

**Pull request merged**